### PR TITLE
Fixed up tests to match new way that crypto stores are created

### PR DESCRIPTION
**Pull request by foldleft**

### CODE REVIEW

File: src/crypto/store/indexeddb-crypto-store.js [hunk 1]
```diff
@@ -314,8 +314,7 @@ export class IndexedDBCryptoStore {
      * @param {string} type A key type
      */
     async getCrossSigningPrivateKey(txn, func, type) {
-        const backend = await this._connect();
-        return backend.getCrossSigningPrivateKey(txn, func, type);
+        this._backend.getCrossSigningPrivateKey(txn, func, type);
```
Comment (jryans) [inline src/crypto/store/indexeddb-crypto-store.js]: Needs `return` I think?

Comment (foldleft) [reply src/crypto/store/indexeddb-crypto-store.js]: oop

File: src/crypto/store/indexeddb-crypto-store.js [hunk 2]
```diff
@@ -314,7 +314,7 @@ export class IndexedDBCryptoStore {
      * @param {string} type A key type
      */
     async getCrossSigningPrivateKey(txn, func, type) {
-        this._backend.getCrossSigningPrivateKey(txn, func, type);
+        return this._backend.getCrossSigningPrivateKey(txn, func, type);
```
Comment (jryans) [inline src/crypto/store/indexeddb-crypto-store.js]: Oh, I think I misunderstood how these work...? Looks like the actual data is passed to `func`, so perhaps the `return` wasn't needed before or now...?

Comment (jryans) [reply src/crypto/store/indexeddb-crypto-store.js]: Also I guess this could drop `async` now...?

Comment (foldleft) [reply src/crypto/store/indexeddb-crypto-store.js]: mm, I suppose it could

**Pull request merged**